src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1177,11 +1249,48 @@ class NodeConfig(object):
                             convert_size(entries[3] + entries[4], 'kb')
                     elif entries[2] == "HugePages_Total:":
                         numa_nodes[num]['HugePages_Total'] = entries[3]
+        # Spread memory resources across NUMA nodes
+        if self.cfg['vnode_per_numa_node'] and len(numa_nodes) > 0:
+            # Huge page memory
+            host_hpmem = self.get_hpmem_on_node(ignore_reserved=True)
+            host_resv_hpmem = host_hpmem - self.get_hpmem_on_node()
+            host_resv_hpmem = 0 if (host_resv_hpmem < 0) else host_resv_hpmem
```

Comment:
This is Python's notion of a ternary operator. Thought it was better than splitting it into two lines.

Reply:
Python's ternary operator can be just the if side.  I'm not suggesting you split this into two lines, I'm just suggesting you lop off the 'else' since it doesn't do anything.  You are saying, set it to 0 or leave it alone.  So just say set it to 0 if host_resv_hpmem < 0.  It'll be left alone if the if condition fails.

Reply:
What you are suggesting produces a syntax error in Python. I'll make the assignment after the comparison.